src/viam/sdk/robot/client.hpp

```diff
@@ -23,13 +25,62 @@ namespace viam {
 namespace sdk {
 
 using grpc::Channel;
-using viam::common::v1::ResourceName;
-using viam::common::v1::Transform;
-using viam::robot::v1::FrameSystemConfig;
 using viam::robot::v1::RobotService;
-using viam::robot::v1::Status;
+using time_point = std::chrono::time_point<long long, std::chrono::nanoseconds>;
```

Comment:
This is a duplicate from `utils.hpp`, which is already being included in this file. I think we can probably get rid of it.

Reply:
When I don't have this line, it gives me errors. 
```
/Users/nicolejung/viam/viam-cpp-sdk/src/viam/sdk/../../viam/sdk/robot/client.hpp:67:21: error: use of undeclared identifier 'time_point'; did you mean 'std::chrono::time_point'?
    boost::optional<time_point> last_reconfigured;
                    ^~~~~~~~~~
                    std::chrono::time_point
```

Reply:
Oh whoops! My mistake, I recall the issue here. `time_point` is `typedef`ed in `utils.*cpp*` so of course it's not accessible in `client.hpp`. There was some back and forth on whether the `typedef` was worthwhile that got stalled on fixing the `time_point` type in general (the fact that the first arg is a `long long` is strange).

We should revisit and fix the `time_point` logic in general. I should've created a ticket for this a while ago; h[ere's one now](https://viam.atlassian.net/browse/RSDK-6574). In the meantime, I think we should avoid having this `using` line in the `.hpp` file entirely, and just give the full type annotation. We can still have a `using` line in the `.cpp` file for ease of use.